.circleci/config.yml

```diff
@@ -13,3 +13,5 @@ jobs:
       - run:
           name: Execute tests
           command: npm test
+      - run: npm install
+      - run: npm run test-hint
```

Comment:
I don't know if we should do this on CI or not.  I lean towards 'no' and let people run it locally.